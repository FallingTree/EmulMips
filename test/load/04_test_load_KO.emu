#-------
#TEST_RETURN_CODE=FAIL          
#TEST_COMMENT="Test d'ouverture d'un fichier"
#-------

load boucle    .o
