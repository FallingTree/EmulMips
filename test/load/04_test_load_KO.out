La commande s'est exécutée correctement
La commande s'est exécutée correctement
La commande s'est exécutée correctement
La commande s'est exécutée correctement
La commande s'est exécutée correctement
La commande load prend en parametre un fichier elf
